src/ralph/assets/utils.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from django.db import transaction
+
+
+def move_parents_models(from_obj, to_obj):
+    """
+    Move parents Django model to diffrent model inheriting from
+    the same parent.
+
+    By default Django does not allow the remove of the model without parents.
+    Clean _meta.parents, we remove object and add back default parents to class
+
+    Args:
+        from_obj: From Django model object
+        to_obj: To Django modelo object
+
+    Example:
+        >> instance = DataCenterAsset.objects.get(pk=1)
+        >> back_office_asset = BackOfficeAsset()
+        >> move_parents_models(instance, back_office_asset)
+    """
+    with transaction.atomic():
+        to_obj.__dict__.update(from_obj.__dict__)
```

Comment:
what about content type of "final" object - is it properly handled here? https://github.com/allegro/ralph/blob/ng/src/ralph/lib/polymorphic/models.py#L176